### PR TITLE
[ELY-1756] [ELY-1757] Fix Elytron-Tool Test Failures on Windows

### PR DESCRIPTION
**Pull request by justinmcook**
Jira: https://issues.jboss.org/browse/ELY-1756
Jira: https://issues.jboss.org/browse/ELY-1757

### CODE REVIEW

File: tool/src/test/java/org/wildfly/security/tool/FileSystemRealmCommandTest.java [hunk 1]
```diff
@@ -247,25 +221,62 @@ private void compareScriptOutput(String fileSystemRealmName, String securityDoma
         assertEquals(scriptLines.get(1), expectedSecurityDomainScript.toString());
     }
 
+    @BeforeClass
+    public static void getAbsolutePaths() throws Exception {
+        String OS = System.getProperty("os.name").toLowerCase();
+        absoluteBaseDir = Paths.get(FileSystemRealmCommandTest.class.getProtectionDomain().getCodeSource().getLocation().toURI()).toAbsolutePath().toString();
+        if (OS.contains("win")) {
+            absoluteBaseDir += "\\filesystem-realm\\";
```
Comment (fjuma) [inline tool/src/test/java/org/wildfly/security/tool/FileSystemRealmCommandTest.java]: Maybe use File.separator here

Comment (justinmcook) [reply tool/src/test/java/org/wildfly/security/tool/FileSystemRealmCommandTest.java]: Changed it to use System.getProperty("file.separator")

File: tool/src/test/java/org/wildfly/security/tool/FileSystemRealmCommandTest.java [hunk 2]
```diff
@@ -247,6 +221,41 @@ private void compareScriptOutput(String fileSystemRealmName, String securityDoma
...
     }
 
+    @BeforeClass
+    public static void getAbsolutePaths() throws Exception {
+        ABSOLUTE_BASE_DIR = Paths.get(FileSystemRealmCommandTest.class.getProtectionDomain().getCodeSource().getLocation().toURI())
+                .toAbsolutePath()
+                + System.getProperty("file.separator")
+                + "filesystem-realm"
+                + System.getProperty("file.separator");
+        ABSOLUTE_BASE_DIR_USERS = ABSOLUTE_BASE_DIR + "users/";
```
Comment (fjuma) [inline tool/src/test/java/org/wildfly/security/tool/FileSystemRealmCommandTest.java]: System.getProperty("file.separator")

File: tool/src/test/java/org/wildfly/security/tool/FileSystemRealmCommandTest.java [hunk 3]
```diff
@@ -247,6 +221,41 @@ private void compareScriptOutput(String fileSystemRealmName, String securityDoma
...
+        ABSOLUTE_BASE_DIR_ROLES = ABSOLUTE_BASE_DIR + "roles/";
```
Comment (fjuma) [inline tool/src/test/java/org/wildfly/security/tool/FileSystemRealmCommandTest.java]: Same here

File: tool/src/test/java/org/wildfly/security/tool/FileSystemRealmCommandTest.java (hunk 2 above)
Comment (justinmcook) [reply tool/src/test/java/org/wildfly/security/tool/FileSystemRealmCommandTest.java]: Fixed

File: tool/src/test/java/org/wildfly/security/tool/FileSystemRealmCommandTest.java (hunk 3 above)
Comment (justinmcook) [reply tool/src/test/java/org/wildfly/security/tool/FileSystemRealmCommandTest.java]: Fixed

**Pull request merged**